components/commons/Loading.tsx

```diff
@@ -1,7 +1,7 @@
 const LoadingPage: React.FC = () => {
   return (
-    <div className="flex items-center justify-center h-screen">
-      <div className="animate-spin rounded-full h-8 w-8 border-t-2 border-b-2 border-primary"></div>
+    <div role="div" className="flex items-center justify-center h-screen">
```

Comment:
can you clarify why adding role of div as "div" is needed?

Reply:
i want to test whether the 2 divs are working properly 
i use getallbyrole to grab them for the testing